specialMenu.php

```diff
@@ -6,7 +6,7 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1" />
 	<title>Specials</title>
 	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous" />
-	<link rel="stylesheet" type="text/css" href="css/specials.css" />
+	<link rel="stylesheet" type="text/css" href="css/main.css" />
 </head>
```

Comment:
The header is not static in position on this page. It disappears when scrolling. This will likely resolve when with the common header file.